scripts/execute_ktlint_for_changed_modules.sh

```diff
@@ -0,0 +1,61 @@
+# This script first finds all the modules changed in a PR then executes ktlint on these modules.
+echo -----Fetching orign master
+git fetch origin master:refs/remotes/origin/master
+git branch -a
+echo -----Done fetching orign master
+
+echo -----Calculating modules to test
+# directory names that corresponds to a module that has unit tests - this list needs to be manually updated when a new module is added/deleted
+TESTABLE_MODULES="payments payments-core paymentsheet wechatpay link stripecardscan identity stripe-core payments-ui-core camera-core"
+# a function to check if a dir is in TESTABLE_MODULES
+isTestableModule() {
+  [[ $TESTABLE_MODULES =~ (^|[[:space:]])$1($|[[:space:]]) ]]
+}
+
+# Files or directories that trigger all the unitest run if changed
```

Comment:
```suggestion
# Files or directories that trigger all the unit tests if changed
```

Reply:
combined the two scripts into one and removed this part